# Data

* [Protein Structure Prediction Center](http://predictioncenter.org/)
    * initiates the CASP experiments to challenge researchers to predict protein structures.
    * contains data on proteins. Contains PDB files.

* [CAMEO](https://www.cameo3d.org/)
    * a place for evaluating the accuracy of protein predictions

* [Dunbrack Lab](http://dunbrack.fccc.edu/PISCES.php)
    * great for getting a list of PDB IDs with certain conditions (like within a resolution cutoff)

* [RCSB](https://www.rcsb.org/#Subcategory-download_structures)
    * for downloading PDB files





# Wiki Pages

* [Protein secondary structure](https://en.wikipedia.org/wiki/Protein_secondary_structure)

* [Multiple sequence alignment](https://en.wikipedia.org/wiki/Multiple_sequence_alignment)

* [CASP](https://en.wikipedia.org/wiki/CASP)





# Biology Web Servers For Predictions

* [RaptorX](http://raptorx.uchicago.edu/)
    * predict protein structures and functions

* [JPred 4](http://www.compbio.dundee.ac.uk/jpred/)
    * predicts secondary structure from PDB sequence

* [CONFOLD](http://protein.rnet.missouri.edu/confold/)
    * predict protein coordinates with contact map, amino acid sequence, and secondary structure

# Programs

* [CNS Solve](http://cns-online.org/v1.3/)
    * used to determine macromolecular structure

* [VMD](https://www.ks.uiuc.edu/Research/vmd/)
    * for visualizing molecules

* [CMView](http://www.bioinformatics.org/cmview/manual.html)
    * for visualizing and analyzing network of contacts between residues

* [DSSP](https://swift.cmbi.umcn.nl/gv/dssp/)
    * Finds secondary structure of existing PDB files
    * does not do any prediction

* [PyMol](https://pymol.org/2/)
    * for visualizing molecules 

* [Modeller](https://salilab.org/modeller/)
    * for visualizing molecules




# Python Libraries
* [Biopython](https://biopython.org/wiki/Documentation)
* [TensorFlow 2.0 Alpha](https://www.tensorflow.org/install)
* [ProDy](http://prody.csb.pitt.edu/)


# Github Links

Here are all the github links that I looked at for this project. Not all the links were useful.

* [ProteinFoldClusters](https://github.com/carlosmartinezvillar/ProteinFoldClusters)

* [DeepMind-alphafold-repl](https://github.com/llSourcell/DeepMind-alphafold-repl)

* [Deep-learning-for-contact_map_v2](https://github.com/doubleQ2018/Deep-learning-for-contact_map_v2)

* [BMI219-2017-ProteinFolding](https://github.com/pfnet-research/BMI219-2017-ProteinFolding)

* [IJCAI2016](https://github.com/icemansina/IJCAI2016)





# Other

* [PROTEIN TERTIARY STRUCTURE](https://molbiol-tools.ca/Protein_tertiary_structure.htm)
    * has links to websites that do tertiary structure prediction
* Reading Code:
    * [How to read code without ripping your hair out](https://medium.com/launch-school/how-to-read-source-code-without-ripping-your-hair-out-e066472bbe8d)
    * [How to quickly and effectively read other peopleâ€™s code](https://selftaughtcoders.com/how-to-quickly-and-effectively-read-other-peoples-code/)